gammapy/stats/variability.py

```diff
@@ -228,3 +229,65 @@ def compute_flux_doubling(flux, flux_err, coords, axis=0):
     }
 
     return doubling_dict
+
+
+def TimmerAlgorithm(power_spectrum, even_freq=True):
```

Comment:
I would take the number of points and the slope of the power spectrum as input. `even_freq` then becomes redundant

Reply:
As expressed above I feel that passing an array representing the periodogram is sometimes helpful - I can add alternative inputs based on your suggestions. I agree that passing the number of points in this case becomes necessary if the input is analytical, and more compact/elegant also in the case of discrete input.